### ISSUE TITLE
Fatal Lua Error at mission start.

### ISSUE THREAD

**Issue by jtyle6** [OpenRA/OpenRA#11643]
I think it's similar Issues as #11383 But I don't know.

````
Fatal Lua Error: 
Stack Traceback
===============
(2) global C function 'error'
(3) Lua function '(anonymous)' at file '/Users/JoelTyler/Documents/OpenRA//lua/sandbox.lua:150' (best guess)
	Local variables:
	 arg = nil
	 ok = boolean: false
	 result = string: "[string \"BindingSupport.lua\"]:30: Uncaught CLR exception at Lua->CLR boundary: System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation. ---> System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation. ---> Eluant.LuaException: [string \"BindingSupport.lua\"]:30: Uncaught CLR exception at Lua->CLR boundary: System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation. ---> System.InvalidOperationException: Actor does not have trait of type `OpenRA.Mods.Common.Traits.PlayerExperience`\
  at OpenRA.TraitDictionary+TraitContainer`1[T].Get (UInt32 actor) <0x151d2100 + 0x000bf> in <filename unknown>:0 \
  at OpenRA.TraitDictionary.Get[T] (OpenRA.Actor actor) <0x151d1f88 + 0x00043> in <filename unknown>:0 \
  at OpenRA.Actor.Trait[T] () <0x151d1f40 + 0x00037> in <filename unknown>:0 \
  at OpenRA.Mods.Common.Scripting.PlayerExperienceProperties..ctor (OpenRA.Scripting.ScriptContext context, OpenRA.Player player) <0x251b1978 + 0x00037> in <filename unknown>:0 \
  at (wrapper managed-to-native) System.Reflection.MonoCMethod:InternalInvoke (System.Reflection.MonoCMethod,object,object[],System.Exception&)\
  at System.Reflection.MonoCMethod.InternalInvoke (System.Object obj, System.Object[] parameters) <0x600bc0 + 0x00033> in <filename unknown>:0 \
  --- End of inner exception stack trace ---\
  at System.Reflection.MonoCMethod.InternalInvoke (System.Object obj, System.Object[] parameters) <0x600bc0 + 0x0007b> in <filename unknown>:0 \
  at System.Reflection.MonoCMethod.DoInvoke (System.Object obj, BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) <0x1d62f58 + 0x000c3> in <filename unknown>:0 \
  at System.Reflection.MonoCMethod.Invoke (BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) <0x1d62f10 + 0x00037> in <filename unknown>:0 \
  at System.Reflection.ConstructorInfo.Invoke (System.Object[] parameters) <0x1d62eb0 + 0x00054> in <filename unknown>:0 \
  at OpenRA.Scripting.ScriptPlayerInterface+<ScriptPlayerInterface>c__AnonStorey0.<>m__0 (System.Type cg) <0x251b17e8 + 0x00087> in <filename unknown>:0 \
  at System.Linq.Enumerable+WhereSelectArrayIterator`2[TSource,TResult].MoveNext () <0xa33e308 + 0x00085> in <filename unknown>:0 \
  at OpenRA.Scripting.ScriptObjectWrapper.Bind (IEnumerable`1 clrObjects) <0x17936740 + 0x00214> in <filename unknown>:0 \
  at OpenRA.Scripting.ScriptPlayerInterface..ctor (OpenRA.Scripting.ScriptContext context, OpenRA.Player player) <0x251b1648 + 0x00183> in <filename unknown>:0 \
  at OpenRA.Player+<OnScriptBind>c__AnonStorey4.<>m__0 () <0x251b15e0 + 0x00047> in <filename unknown>:0 \
  at System.Lazy`1[T].CreateValue () <0x1a87500 + 0x001ab> in <filename unknown>:0 \
  at Eluant.LuaRuntime.Call (IList`1 args) <0x17931b78 + 0x0023b> in <filename unknown>:0 \
  at Eluant.LuaRuntime.Call (Eluant.LuaFunction fn, IList`1 args) <0x179361d8 + 0x0004f> in <filename unknown>:0 \
  at Eluant.LuaFunction.Call (IList`1 args) <0x179361a0 + 0x00023> in <filename unknown>:0 \
  at Eluant.LuaFunction.Call (Eluant.LuaValue[] args) <0x17936178 + 0x00017> in <filename unknown>:0 \
  at OpenRA.Mods.Common.Scripting.UtilsGlobal.Do (Eluant.LuaValue[] collection, Eluant.LuaFunction func) <0x17941558 + 0x0006f> in <filename unknown>:0 \
  at (wrapper managed-to-native) System.Reflection.MonoMethod:InternalInvoke (System.Reflection.MonoMethod,object,object[],System.Exception&)\
  at System.Reflection.MonoMethod.Invoke (System.Object obj, BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) <0x1abae08 + 0x00093> in <filename unknown>:0 \
  --- End of inner exception stack trace ---\
  at System.Reflection.MonoMethod.Invoke (System.Object obj, BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) <0x1abae08 + 0x000e7> in <filename unknown>:0 \
  at System.Reflection.MethodBase.Invoke (System.Object obj, System.Object[] parameters) <0x1abadc8 + 0x00039> in <filename unknown>:0 \
  at OpenRA.Scripting.ScriptMemberWrapper.Invoke (Eluant.LuaVararg args) <0x1793e700 + 0x003ff> in <filename unknown>:0 \
  at (wrapper managed-to-native) System.Reflection.MonoMethod:InternalInvoke (System.Reflection.MonoMethod,object,object[],System.Exception&)\
  at System.Reflection.MonoMethod.Invoke (System.Object obj, BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) <0x1abae08 + 0x00093> in <filename unknown>:0 \
  --- End of inner exception stack trace ---\
  at System.Reflection.MonoMethod.Invoke (System.Object obj, BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) <0x1abae08 + 0x000e7> in <filename unknown>:0 \
  at System.Reflection.MethodBase.Invoke (System.Object obj, System.Object[] parameters) <0x1abadc8 + 0x00039> in <filename unknown>:0 \
  at Eluant.LuaRuntime+MethodWrapper.Invoke (System.Object[] parms) <0x1793e6d0 + 0x00027> in <filename unknown>:0 \
  at Eluant.LuaRuntime.MakeManagedCall (IntPtr state, Eluant.MethodWrapper wrapper) <0x1793d000 + 0x00fa3> in <filename unknown>:0 "
(4) tail call
(5) Lua function '?' at line 15 of chunk '"scriptwrapper.lua"]'
(6) global C function 'xpcall'
(7) Lua upvalue 'TryRunSandboxed' at line 15 of chunk '"scriptwrapper.lua"]'
	Local variables:
	 fn = Lua function '?' (defined at line 302 of chunk "evacuation.lua"])
(8) Lua function '?' at line 23 of chunk '"scriptwrapper.lua"]'

Error message
===============
/Users/JoelTyler/Documents/OpenRA//lua/sandbox.lua:150: [string "BindingSupport.lua"]:30: Uncaught CLR exception at Lua->CLR boundary: System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation. ---> System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation. ---> Eluant.LuaException: [string "BindingSupport.lua"]:30: Uncaught CLR exception at Lua->CLR boundary: System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation. ---> System.InvalidOperationException: Actor does not have trait of type `OpenRA.Mods.Common.Traits.PlayerExperience`
  at OpenRA.TraitDictionary+TraitContainer`1[T].Get (UInt32 actor) <0x151d2100 + 0x000bf> in <filename unknown>:0 
  at OpenRA.TraitDictionary.Get[T] (OpenRA.Actor actor) <0x151d1f88 + 0x00043> in <filename unknown>:0 
  at OpenRA.Actor.Trait[T] () <0x151d1f40 + 0x00037> in <filename unknown>:0 
  at OpenRA.Mods.Common.Scripting.PlayerExperienceProperties..ctor (OpenRA.Scripting.ScriptContext context, OpenRA.Player player) <0x251b1978 + 0x00037> in <filename unknown>:0 
  at (wrapper managed-to-native) System.Reflection.MonoCMethod:InternalInvoke (System.Reflection.MonoCMethod,object,object[],System.Exception&)
  at System.Reflection.MonoCMethod.InternalInvoke (System.Object obj, System.Object[] parameters) <0x600bc0 + 0x00033> in <filename unknown>:0 
  --- End of inner exception stack trace ---
  at System.Reflection.MonoCMethod.InternalInvoke (System.Object obj, System.Object[] parameters) <0x600bc0 + 0x0007b> in <filename unknown>:0 
  at System.Reflection.MonoCMethod.DoInvoke (System.Object obj, BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) <0x1d62f58 + 0x000c3> in <filename unknown>:0 
  at System.Reflection.MonoCMethod.Invoke (BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) <0x1d62f10 + 0x00037> in <filename unknown>:0 
  at System.Reflection.ConstructorInfo.Invoke (System.Object[] parameters) <0x1d62eb0 + 0x00054> in <filename unknown>:0 
  at OpenRA.Scripting.ScriptPlayerInterface+<ScriptPlayerInterface>c__AnonStorey0.<>m__0 (System.Type cg) <0x251b17e8 + 0x00087> in <filename unknown>:0 
  at System.Linq.Enumerable+WhereSelectArrayIterator`2[TSource,TResult].MoveNext () <0xa33e308 + 0x00085> in <filename unknown>:0 
  at OpenRA.Scripting.ScriptObjectWrapper.Bind (IEnumerable`1 clrObjects) <0x17936740 + 0x00214> in <filename unknown>:0 
  at OpenRA.Scripting.ScriptPlayerInterface..ctor (OpenRA.Scripting.ScriptContext context, OpenRA.Player player) <0x251b1648 + 0x00183> in <filename unknown>:0 
  at OpenRA.Player+<OnScriptBind>c__AnonStorey4.<>m__0 () <0x251b15e0 + 0x00047> in <filename unknown>:0 
  at System.Lazy`1[T].CreateValue () <0x1a87500 + 0x001ab> in <filename unknown>:0 
  at Eluant.LuaRuntime.Call (IList`1 args) <0x17931b78 + 0x0023b> in <filename unknown>:0 
  at Eluant.LuaRuntime.Call (Eluant.LuaFunction fn, IList`1 args) <0x179361d8 + 0x0004f> in <filename unknown>:0 
  at Eluant.LuaFunction.Call (IList`1 args) <0x179361a0 + 0x00023> in <filename unknown>:0 
  at Eluant.LuaFunction.Call (Eluant.LuaValue[] args) <0x17936178 + 0x00017> in <filename unknown>:0 
  at OpenRA.Mods.Common.Scripting.UtilsGlobal.Do (Eluant.LuaValue[] collection, Eluant.LuaFunction func) <0x17941558 + 0x0006f> in <filename unknown>:0 
  at (wrapper managed-to-native) System.Reflection.MonoMethod:InternalInvoke (System.Reflection.MonoMethod,object,object[],System.Exception&)
  at System.Reflection.MonoMethod.Invoke (System.Object obj, BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) <0x1abae08 + 0x00093> in <filename unknown>:0 
  --- End of inner exception stack trace ---
  at System.Reflection.MonoMethod.Invoke (System.Object obj, BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) <0x1abae08 + 0x000e7> in <filename unknown>:0 
  at System.Reflection.MethodBase.Invoke (System.Object obj, System.Object[] parameters) <0x1abadc8 + 0x00039> in <filename unknown>:0 
  at OpenRA.Scripting.ScriptMemberWrapper.Invoke (Eluant.LuaVararg args) <0x1793e700 + 0x003ff> in <filename unknown>:0 
  at (wrapper managed-to-native) System.Reflection.MonoMethod:InternalInvoke (System.Reflection.MonoMethod,object,object[],System.Exception&)
  at System.Reflection.MonoMethod.Invoke (System.Object obj, BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) <0x1abae08 + 0x00093> in <filename unknown>:0 
  --- End of inner exception stack trace ---
  at System.Reflection.MonoMethod.Invoke (System.Object obj, BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) <0x1abae08 + 0x000e7> in <filename unknown>:0 
  at System.Reflection.MethodBase.Invoke (System.Object obj, System.Object[] parameters) <0x1abadc8 + 0x00039> in <filename unknown>:0 
  at Eluant.LuaRuntime+MethodWrapper.Invoke (System.Object[] parms) <0x1793e6d0 + 0x00027> in <filename unknown>:0 
  at Eluant.LuaRuntime.MakeManagedCall (IntPtr state, Eluant.MethodWrapper wrapper) <0x1793d000 + 0x00fa3> in <filename unknown>:0 
===============
  at OpenRA.Scripting.ScriptContext.FatalError (System.String message) [0x00000] in <filename unknown>:0 
  at System.Reflection.MonoMethod.InternalInvoke (System.Reflection.MonoMethod , System.Object , System.Object[] , System.Exception& ) [0x00000] in <filename unknown>:0 
  at System.Reflection.MonoMethod.Invoke (System.Object obj, BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) [0x00000] in <filename unknown>:0 
  at System.Reflection.MethodBase.Invoke (System.Object obj, System.Object[] parameters) [0x00000] in <filename unknown>:0 
  at Eluant.LuaRuntime+MethodWrapper.Invoke (System.Object[] parms) [0x00000] in <filename unknown>:0 
  at Eluant.LuaRuntime.MakeManagedCall (IntPtr state, Eluant.MethodWrapper wrapper) [0x00000] in <filename unknown>:0 
  at Eluant.LuaRuntime.MethodWrapperCallCallback (IntPtr state) [0x00000] in <filename unknown>:0 
  at Eluant.LuaRuntime.MethodWrapperCallCallbackWrapper (IntPtr state) [0x00000] in <filename unknown>:0 
  at Eluant.LuaApi.lua_pcall (IntPtr , Int32 , Int32 , Int32 ) [0x00000] in <filename unknown>:0 
  at Eluant.LuaRuntime.Call (IList`1 args) [0x00000] in <filename unknown>:0 
  at Eluant.LuaRuntime.Call (Eluant.LuaFunction fn, IList`1 args) [0x00000] in <filename unknown>:0 
  at Eluant.LuaFunction.Call (IList`1 args) [0x00000] in <filename unknown>:0 
  at Eluant.LuaFunction.Call (Eluant.LuaValue[] args) [0x00000] in <filename unknown>:0 
  at OpenRA.Scripting.ScriptContext.WorldLoaded () [0x00000] in <filename unknown>:0 
  at OpenRA.Mods.Common.Scripting.LuaScript.WorldLoaded (OpenRA.World world, OpenRA.Graphics.WorldRenderer worldRenderer) [0x00000] in <filename unknown>:0 
  at OpenRA.World.LoadComplete (OpenRA.Graphics.WorldRenderer wr) [0x00000] in <filename unknown>:0 
  at OpenRA.Game.StartGame (System.String mapUID, WorldType type) [0x00000] in <filename unknown>:0 
  at OpenRA.Network.UnitOrders.ProcessOrder (OpenRA.Network.OrderManager orderManager, OpenRA.World world, Int32 clientId, OpenRA.Order order) [0x00000] in <filename unknown>:0 
  at OpenRA.Network.OrderManager.TickImmediate () [0x00000] in <filename unknown>:0 
  at OpenRA.Sync+<CheckSyncUnchanged>c__AnonStorey0.<>m__0 () [0x00000] in <filename unknown>:0 
  at OpenRA.Sync.CheckSyncUnchanged[T] (OpenRA.World world, System.Func`1 fn) [0x00000] in <filename unknown>:0 
  at OpenRA.Sync.CheckSyncUnchanged (OpenRA.World world, System.Action fn) [0x00000] in <filename unknown>:0 
  at OpenRA.Game.InnerLogicTick (OpenRA.Network.OrderManager orderManager) [0x00000] in <filename unknown>:0 
  at OpenRA.Game.LogicTick () [0x00000] in <filename unknown>:0 
  at OpenRA.Game.Loop () [0x00000] in <filename unknown>:0 
  at OpenRA.Game.Run () [0x00000] in <filename unknown>:0 
  at OpenRA.Program.Run (System.String[] args) [0x00000] in <filename unknown>:0 
  at OpenRA.Program.Main (System.String[] args) [0x00000] in <filename unknown>:0 
 ```

**Comment by GraionDilach** [comment]
Nope, this is a recent bleed regression, with #11629 the fix for it.

**Comment by jtyle6** [comment]
Ah,  ok. I like to pull latest build down.

**Comment by GraionDilach** [comment]
This is fixed now with #11629 merged. Requesting someone with rights to close this.
